CONTRIBUTING.md

```diff
@@ -58,42 +61,53 @@ go build
 **Running the linter**:
 
 We make use of the [golangci-lint](https://github.com/golangci/golangci-lint) tool to lint the code in CI. To run it locally, first install it:
+
 ```bash
 go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
 ```
+
 then run:
-```
-golangci-lint run --out-format=tab --new-from-rev master ./...
+
+```bash
+make lint
 ```
 
-If you've added new dependencies you might also want to check and make sure all dependencies exists in `vendor/` folder by running:
+You can also run the linter inside the docker container, which will benefit from the version of the linter being the same as it will be in CI.
+
 ```bash
-go get -u github.com/FiloSottile/vendorcheck
-vendorcheck ./...
+make ci-like-lint
 ```
 
 **Running the test suite**:
 
 To exercise the entire test suite:
+
 ```bash
-go test -race ./...
+make test
```

Comment:
We had a conversation the other day with @imiric, and he provided me a comment which seems reasonable:

> it's a test process, not a "tests" artifact; same as "lint," instead of "linters"

Does that make sense? :)